recipes/jbig/Makefile.patch

```diff
@@ -0,0 +1,11 @@
+--- Makefile	2014-03-27 14:47:15.000000000 -0400
++++ Makefile	2015-11-06 14:14:48.000000000 -0500
+@@ -4,7 +4,7 @@
+ CC = gcc
+ 
+ # Options for the compiler: A high optimization level is suggested
+-CFLAGS = -O2 -W -Wno-unused-result
++CFLAGS = -O2 -W -fPIC
```

Comment:
also, it's really, really bad to set CFLAGS without respecting any existing setting.  Please always do

    CFLAGS = $CFLAGS <+ additions>

For example, one of the things this breaks is the multilib 32-bit building.